pkg/harvester/components/novnc/NovncConsole.vue

```diff
@@ -13,6 +13,15 @@
         </div>
       </main>
     </div>
+    <div v-if="reconnecting">
+      <main class="main-layout">
+        <div class="text-center">
+          <h2 class="text-secondary mt-20">
+            {{ t('vncConsole.reconnecting.message') }} ...  {{ reconnectingTimes }} times
```

Comment:
I would slightly change the message (adding the maximum number of attempts eventually):
```
Web VNC reconnection attempt: 1 of 10
```

Reply:
@torchiaf  Good advice, it make me come up with other way. How about we just add loading icon?